admin-manual/security/security.rst

```diff
@@ -207,6 +207,39 @@ Shibboleth backend configuration in Storage Service
 
 #. Restart the Storage Service.
 
+.. _oidc-setup:
+
+OIDC setup
+==========
+
+`OIDC <oidc_>`_ (OpenID Connect) is an identity layer on top of OAuth 2.0, providing
+identity verification, authentication and basic information about the end user. This
+feature allows administrators to deploy both the Archivematica Dashboard and Storage
+Service Django applications with Shibboleth authentication.
```

Comment:
Is it Shibboleth auth?

Reply:
Copy paste error on my part - that should say OIDC :-)